omero/developers/whatsnew.txt

```diff
@@ -36,6 +36,9 @@ further details.
 OMERO.web template and view writers need to use the ``@render_response()``
 decorator, see :ref:`render-response` for more details.
 
+The permissions attribute on data manager has also been cleaned up, removing
+`-locked`.
```

Comment:
There is no need to mention about removal as that was removed long time ago. I just cleaned up few places it was missed